config/environment.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV.meetingsUrl = 'http://localhost:8000';
```

Comment:
I'm not sure if I understand exactly, but when every we are in development mode (running locally), meetingsUrl will refer to `http://localhost:8000`. We can make test and production mode point to staging and osf.io respectively.